setup.py

```diff
@@ -31,17 +31,20 @@
         saq=saq.__main__:main
     """,
     install_requires=[
-        "redis>=4.2,<6.0",
         "croniter>=0.3.18",
     ],
     extras_require={
         "hiredis": ["redis[hiredis]>=4.2.0"],
+        "postgres": ["psycopg[pool]"],
+        "redis": ["redis>=4.2,<6.0"],
         "web": ["aiohttp", "aiohttp_basicauth"],
         "dev": [
             "black",
             "coverage",
             "mypy",
+            "psycopg[pool]",
```

Comment:
We should just update the `make deps` command to include redis and postgres: https://github.com/tobymao/saq/blob/master/Makefile#L2